config.json

```diff
@@ -76,6 +76,7 @@
       "maxRetries": 3,
       "retryDelay": 10000,
       "modifiedThreshold": 100,
+      "chunkUploadSize": 1,
```

Comment:
I realize this is only the standalone server's configuration, but I think it would be beneficial to make the default setting larger than 1 mb. I'm just thinking of how many chunks we'd have to upload for a 1 gb file. Maybe a default of 50mb - 100mb? What are your thoughts?

Reply:
I think so too. We are going to run through Eric to set a proper value on the lib-smb/Config.json